packages/makeup-modal/src/index.js

```diff
@@ -20,9 +75,13 @@ function unmodal() {
     return modalEl;
 }
 
-function modal(el, options) {
+function modal(el, options = {}) {
     unmodal();
     modalEl = el;
+    debugger;
+    if (options.hoist) {
```

Comment:
Can we stick with the same pattern we have elsewhere for options, where the defaultOptions are explicit:

https://github.com/makeup/makeup-js/blob/master/packages/makeup-listbox/src/index.js#L25

Reply:
Could you give a more specific example of what that would look like in this case? I don’t understand.

Reply:
If you look at this line you can see where the default options are declared:

https://github.com/makeup/makeup-js/blob/master/packages/makeup-listbox/src/index.js#L12

And this line shows how the selected options are merged in:

https://github.com/makeup/makeup-js/blob/master/packages/makeup-listbox/src/index.js#L25

NOTE: this is kind of old syntax now. [Spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_object_literals) is going to make this easier, e.g.:

`this._options = {...defaultOptions, ...selectedOptions}`

But let's stick with the old syntax for now until I go through and update everywhere.

Reply:
Yeah I agree that spread syntax would be good! We could even combine spread, defaults, and object destructuring and do something like 
```
function modal(el, { hoist =  false, ...restOfOptions }){
    if(hoist){hoist(modalEl)}
    keyboardtrap(restOfOptions)
    screenreaderTrap(restOfOptions)
}
```

But yeah we can talk about it when we get there! 👍